runtime/near-vm-runner/src/tests.rs

```diff
@@ -27,6 +28,11 @@ pub(crate) fn with_vm_variants(runner: fn(VMKind) -> ()) {
 
     #[cfg(all(feature = "wasmer2_vm", target_arch = "x86_64"))]
     runner(VMKind::Wasmer2);
+
+    #[cfg(all(feature = "near_vm", target_arch = "x86_64"))]
+    if cfg.limit_config.contract_prepare_version == ContractPrepareVersion::V2 {
```

Comment:
This should probably read `>=` instead of `==` to ensure we continue testing `NearVm` as the contract preparation code evolves.

Reply:
Hmm so I’m not sure I agree. If contract preparation code evolves in ways that require adding a new variant here, then NearVm would also need to evolve. Yet, VMKind does not allow disambiguating for this. So I think if we were to introduce a contract prepare version 3, then we would also need to add a VMKind::NearVm2 to get tests to pass.

This being said, I also think this is all moot anyway, as this is hopefully the last version we’ll have before properly implementing limited replayability, and then we should be able to get rid of the ContractPrepareVersion enum altogether, alongside VMKind::{Wasmer0, Wasmer2} :)